### PR TITLE
Added the links for all the things needed

### PR DESCRIPTION
**Pull request by gautam071**
**Issue** #1 

**Description**
    - All the links required to download in mac os is added
    - Command lines for some
    - App store links have been added.

**Note**
   The Google chat app link is not added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,39 +2,51 @@
 A curated list of utilities we use on mac. This guide will help you to set up your development environment.
 
 ### First things first
-1. Chrome
-2. Brave
+1. [Chrome](https://www.google.com/chrome/?brand=CHBD&gclsrc=ds&gclsrc=ds)
```
Comment (rathorevaibhav) [inline README.md]:
```suggestion
1. [Chrome](https://www.google.com/chrome/)
```

File: README.md [hunk 2]
```diff
@@ -2,39 +2,51 @@
...
+2. [Brave](https://brave.com/download/)
 
 ### Communication
-1. Slack
-2. Discord
+1. [Slack](https://slack.com/intl/en-in/downloads/mac)
+   -AppStore [Slack](https://apps.apple.com/app/slack/id803453959?ls=1&mt=12)
```
Comment (rathorevaibhav) [inline README.md]: we can remove this. there's already a link in their website for app store

File: README.md [hunk 3]
```diff
@@ -2,39 +2,51 @@
...
+2. [Discord macOS 10.11 (El Capitan) or higher](https://discord.com/download)
 3. Chat
```
Comment (rathorevaibhav) [inline README.md]: chat.google.com

File: README.md [hunk 4]
```diff
@@ -2,39 +2,51 @@
...
-2. Sequel Pro
-3. DB Ngine
+1. [TablePlus macOS 10.11+](https://tableplus.com/download)
+2. [Sequel Pro macOS 10.6+](https://sequelpro.com/download)
+3. [DB Ngin macOS 10.11+](https://dbngin.com/)
 
 ### Terminal
-1. iTerm
-2. Hyper
+1. [iTerm OS 10.14+](https://iterm2.com/downloads.html) -Preffered Stable
```
Comment (rathorevaibhav) [inline README.md]:
```suggestion
1. [iTerm OS 10.14+](https://iterm2.com/downloads.html)
```

File: README.md [hunk 5]
```diff
@@ -2,39 +2,51 @@
...
+2. [Hyper](https://hyper.is/)
 3. Oh My ZSH
-4. Powershell 10k
+ - ByWget
```
Comment (rathorevaibhav) [inline README.md]:
```suggestion
 - Using wget 
```

File: README.md [hunk 6]
```diff
@@ -2,39 +2,51 @@
...
-2. Hyper
+1. [iTerm OS 10.14+](https://iterm2.com/downloads.html) -Preffered Stable
+2. [Hyper](https://hyper.is/)
 3. Oh My ZSH
-4. Powershell 10k
+ - ByWget 
+    ```
+        -sh -c "$(wget https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh -O -)"
+    ```
+ - By Curl
```
Comment (rathorevaibhav) [inline README.md]:
```suggestion
 - Using curl
```

File: README.md [hunk 7]
```diff
@@ -2,39 +2,51 @@
...
 
 ### Terminal
-1. iTerm
-2. Hyper
+1. [iTerm OS 10.14+](https://iterm2.com/downloads.html) -Preffered Stable
+2. [Hyper](https://hyper.is/)
 3. Oh My ZSH
-4. Powershell 10k
+ - ByWget 
+    ```
```
Comment (rathorevaibhav) [inline README.md]:
```suggestion
    ```sh
```

File: README.md [hunk 8]
```diff
@@ -2,39 +2,51 @@
...
+        -sh -c "$(wget https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh -O -)"
+    ```
+ - By Curl
+    ```
```
Comment (rathorevaibhav) [inline README.md]:
```suggestion
    ```sh
```

File: README.md [hunk 9]
```diff
@@ -2,39 +2,51 @@
...
+        -sh -c "$(curl -fsSL https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"
+    ``` 
+4. [Powershell 10k](https://github.com/PowerShell/PowerShell/releases/tag/v7.1.4) Go to the bottom of the page you can find the all the links
```
Comment (rathorevaibhav) [inline README.md]:
```suggestion
4. [Powerlevel10k](https://github.com/romkatv/powerlevel10k)
```

File: README.md [hunk 10]
```diff
@@ -2,39 +2,51 @@
...
 
 
 ### Developer Tools
-1. Postman
-2. VS Code
-3. Sublime Text
+1. [Postman](https://www.postman.com/downloads/)
+2. [VS Code](https://code.visualstudio.com/download)
+3. [Sublime Text](sublimetext.com/download)
```
Comment (rathorevaibhav) [inline README.md]: not working

File: README.md [hunk 11]
```diff
@@ -2,39 +2,51 @@
...
+2. [Docker](https://docs.docker.com/desktop/mac/install/)
 
 ### Commands
-1. XCode command line utilities
+1. XCode command line utilities 
     ```
     xcode-select install
     ```
 2. Valet
+    ```
```
Comment (rathorevaibhav) [inline README.md]:
```suggestion
    ```sh
```

File: README.md [hunk 12]
```diff
@@ -2,39 +2,51 @@
...
 ### Development Environment
-1. MAMP
-2. Docker
+1. [MAMP](https://www.mamp.info/en/mamp/mac/)
+2. [Docker](https://docs.docker.com/desktop/mac/install/)
 
 ### Commands
-1. XCode command line utilities
+1. XCode command line utilities 
     ```
```
Comment (rathorevaibhav) [inline README.md]:
```suggestion
    ```sh
```

**Review by rathorevaibhav — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gautam071) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (gautam071) [reply README.md]: Okay

File: README.md (hunk 4 above)
Comment (gautam071) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (gautam071) [reply README.md]: Done

File: README.md (hunk 6 above)
Comment (gautam071) [reply README.md]: Done

File: README.md (hunk 7 above)
Comment (gautam071) [reply README.md]: Added

File: README.md (hunk 8 above)
Comment (gautam071) [reply README.md]: Added

File: README.md (hunk 9 above)
Comment (gautam071) [reply README.md]: Changed

File: README.md (hunk 10 above)
Comment (gautam071) [reply README.md]: Changed and checked its working

File: README.md (hunk 11 above)
Comment (gautam071) [reply README.md]: Done

File: README.md (hunk 12 above)
Comment (gautam071) [reply README.md]: Done

File: README.md [hunk 13]
```diff
@@ -2,39 +2,50 @@
...
+2. [Brave](https://brave.com/download/)
 
 ### Communication
-1. Slack
-2. Discord
-3. Chat
-4. Zoom
+1. [Slack](https://slack.com/intl/en-in/downloads/mac)
+2. [Discord macOS 10.11 (El Capitan) or higher](https://discord.com/download)
+3. [Chat](chat.google.com)
```
Comment (rathorevaibhav) [inline README.md]: not working

Comment (gautam071) [reply README.md]: Changed and checked its working

**Review by rathorevaibhav — APPROVED** [review]

**Pull request merged**